tests/sqlparser_mssql.rs

```diff
@@ -2036,3 +2036,37 @@ fn parse_mssql_merge_with_output() {
         OUTPUT $action, deleted.ProductID INTO dsi.temp_products";
     ms_and_generic().verified_stmt(stmt);
 }
+
+#[test]
+fn parse_print() {
```

Comment:
Do we need a test that asserts a particular error for a bare `PRINT`? Not sure what the conventions are for that